docs/consume-packages/install-use-packages-nuget-cli.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Install and manage NuGet packages using the nuget.exe CLI
+description: Instructions for using the nuget.exe CLI to work with NuGet packages.
+author: mikejo5000
+ms.author: mikejo
+ms.date: 06/03/2019
+ms.topic: conceptual
+---
+
+# Install and manage packages using the nuget.exe CLI
+
+The CLI tool allows you to easily install, uninstall, and update NuGet packages in projects and solutions. This tool provides all NuGet capabilities on Windows, and also provides most features on Mac and Linux when running under Mono.
```

Comment:
The CLI tool cannot install or uninstall NuGet packages in projects and solutions to the best of my knowledge. It can only restore and update packages from projects/solutions. The CLI has no uninstall capability at all. It can "install" (unzip) into a directory, but not install into a project (adding to packages.config and modify the csproj)

Reply:
OK, thanks. I will try to get clarity.

Reply:
The Examples section in the live reference doc shows usage with packages.config and -OutputDirectory, but I'm not sure yet if it does what it is supposed to do:
https://review.docs.microsoft.com/en-us/nuget/tools/cli-ref-install?branch=pr-en-us-1467#examples

Reply:
@karann-msft @nkolev92 The live reference doc on install does seem to agree with Andy's comments. From the live doc on the nuget.exe CLI install command:

<begin>
The install command does not modify a project file or packages.config; in this way it's similar to restore in that it only adds packages to disk but does not change a project's dependencies.

To add a dependency, either add a package through the Package Manager UI or Console in Visual Studio, or modify packages.config and then run either install or restore.
<end>
That's why I used similar text in the earlier draft. Reading through the ref doc, it's unclear to me how one would specify the .csproj in the install command...